.github/workflows/rust.yml

```diff
@@ -55,22 +67,22 @@ jobs:
             rust: stable-x86_64
             target: x86_64-pc-windows-msvc
     steps:
-    - uses: actions/checkout@master
-    - run: git submodule init
-    - run: git submodule update
-    - name: Set up vcpkg on Windows
-      if: startsWith(matrix.os, 'windows') && ! contains(matrix.build, 'vendor')
-      run: |
-        git clone https://github.com/microsoft/vcpkg
-        & .\vcpkg\bootstrap-vcpkg.bat
-        & .\vcpkg\vcpkg install sundials --triplet=x64-windows
-    - name: Install Ubuntu packages
-      if: startsWith(matrix.os, 'ubuntu') && ! contains(matrix.build, 'vendor')
-      run: |
-        sudo apt-get update && sudo apt-get -y install libsundials-dev
-    - name: Install Rust (rustup)
-      run: rustup update ${{ matrix.rust }} --no-self-update && rustup default ${{ matrix.rust }}
-      shell: bash
-    - run: rustup target add ${{ matrix.target }}
-    - run: cargo build
-    - run: cargo test
+      - uses: actions/checkout@master
```

Comment:
Why this whitespace change?

Reply:
autoformatting from Prettier - reverted to the old version